include/ibamr/IBFESurfaceMethod.h

```diff
@@ -529,14 +586,22 @@ class IBFESurfaceMethod : public IBStrategy
     const unsigned int d_num_parts = 1;
     std::vector<IBTK::FEDataManager*> d_fe_data_managers;
     SAMRAI::hier::IntVector<NDIM> d_ghosts = 0;
-    std::vector<libMesh::System*> d_X_systems, d_U_systems, d_U_n_systems, d_U_t_systems, d_F_systems, d_DP_systems;
+    std::vector<libMesh::System*> d_X_systems, d_U_systems, d_U_n_systems, d_U_t_systems, d_F_systems, d_P_jump_systems,
```

Comment:
Please merge this with the next line.

Reply:
Also, suggest using the ordering:

1. positions
2. velocities
3. forces
4. jump conditions
5. post-processed interfacial quantities.

There are a lot of variables now, and I think it will be helpful for others reading the code if things are listed consistently.